packages/react-components/react-table/src/stories/Table/MultipleSelect.stories.tsx

```diff
@@ -121,10 +130,10 @@ export const MultipleSelect = () => {
           <TableHeaderCell>Last update</TableHeaderCell>
         </TableRow>
       </TableHeader>
-      <TableBody>
-        {rows.map(({ item, selected, toggleSelect }) => (
-          <TableRow key={item.file.label} onClick={toggleSelect} aria-selected={selected}>
-            <TableSelectionCell checked={selected} />
+      <TableBody ref={ref}>
```

Comment:
the `mover` is on the body because of a bug in tabster regarding inputs, I'll report this there